onnxruntime/core/providers/webnn/builders/helper.h

```diff
@@ -133,7 +139,11 @@ inline bool CheckSingleOp(const std::string& op_type, const emscripten::val& wnn
   return op_map.find(op_type) != op_map.end() && wnn_builder_[op_map.find(op_type)->second].as<bool>();
 }
 
-constexpr std::array<ONNX_NAMESPACE::TensorProto_DataType, 6> supported_data_types = {
+constexpr std::array<ONNX_NAMESPACE::TensorProto_DataType, 1> supported_cpu_data_types = {
+    ONNX_NAMESPACE::TensorProto_DataType_FLOAT,
+};
+
+constexpr std::array<ONNX_NAMESPACE::TensorProto_DataType, 6> supported_gpu_data_types = {
```

Comment:
Ideally there would be a way to detect this without needing to store tables inside ORT, which could become out of sync (and would fail if running on older versions of Chrome which had less data type support). I don't have a better idea at the moment though. 🤔

Reply:
Btw, my fork also supports uint64 (not just int64), not that it really matters though because ONNX ops use int64 for indices.

Reply:
uint64 added. :)

> Ideally there would be a way to detect this without needing to store tables inside ORT, which could become out of sync (and would fail if running on older versions of Chrome which had less data type support). I don't have a better idea at the moment though. 

Yeah, ideally if WebNN could provide API to detect the supported data type status that would be perfect. Currently we have no other good option.